service/policy/objects.proto

```diff
@@ -430,6 +430,19 @@ message RegisteredResourceValue {
 
   RegisteredResource resource = 3;
 
+  repeated RegisteredResourceActionAttributeValue action_attribute_values = 4;
+
+  // Common metadata
+  common.Metadata metadata = 100;
+}
+
+message RegisteredResourceActionAttributeValue {
```

Comment:
Move this inside of `RegisteredResourceValue` to make it clear this is only used within a RRV.